canto 5.0.0: fail to start with error panic: unknown field "merge_fork_block" in types.ChainConfig

I try to start the canto v5.0.0 from block 0, but fail with the following error
panic: unknown field "merge_fork_block" in types.ChainConfig

Maybe canto v6.0.0 fix it?

--- a/.github/workflows/publish.yml
+++ b/.github/workflows/publish.yml
@@ -33,7 +33,7 @@ jobs:
           - project: bostrom
             version: v0.3.2
           - project: canto
-            version: v5.0.0
+            version: v6.0.0
           - project: cheqd
             version: 0.6.9
           - project: chihuahua

--- a/README.md
+++ b/README.md
@@ -54,7 +54,7 @@ tagged with the form `$COSMOS_OMNIBUS_VERSION-$PROJECT-$PROJECT_VERSION`.
 |[bitcanna](https://github.com/BitCannaGlobal/bcna)|`v2.0.2`|`ghcr.io/akash-network/cosmos-omnibus:v0.3.42-bitcanna-v2.0.2`|[Example](./bitcanna)|
 |[bitsong](https://github.com/bitsongofficial/go-bitsong)|`v0.14.0`|`ghcr.io/akash-network/cosmos-omnibus:v0.3.42-bitsong-v0.14.0`|[Example](./bitsong)|
 |[bostrom](https://github.com/cybercongress/go-cyber)|`v0.3.2`|`ghcr.io/akash-network/cosmos-omnibus:v0.3.42-bostrom-v0.3.2`|[Example](./bostrom)|
-|[canto](https://github.com/Canto-Network/Canto)|`v5.0.0`|`ghcr.io/akash-network/cosmos-omnibus:v0.3.42-canto-v5.0.0`|[Example](./canto)|
+|[canto](https://github.com/Canto-Network/Canto)|`v6.0.0`|`ghcr.io/akash-network/cosmos-omnibus:v0.3.42-canto-v6.0.0`|[Example](./canto)|
 |[cheqd](https://github.com/cheqd/cheqd-node)|`0.6.9`|`ghcr.io/akash-network/cosmos-omnibus:v0.3.42-cheqd-0.6.9`|[Example](./cheqd)|
 |[chihuahua](https://github.com/ChihuahuaChain/chihuahua)|`v4.2.3`|`ghcr.io/akash-network/cosmos-omnibus:v0.3.42-chihuahua-v4.2.3`|[Example](./chihuahua)|
 |[comdex](https://github.com/comdex-official/comdex)|`v11.5.0`|`ghcr.io/akash-network/cosmos-omnibus:v0.3.42-comdex-v11.5.0`|[Example](./comdex)|

--- a/canto/README.md
+++ b/canto/README.md
@@ -2,12 +2,12 @@
 
 | | |
 |---|---|
-|Version|`v5.0.0`|
+|Version|`v6.0.0`|
 |Binary|`cantod`|
 |Directory|`.cantod`|
 |ENV namespace|`CANTOD`|
 |Repository|`https://github.com/Canto-Network/Canto`|
-|Image|`ghcr.io/akash-network/cosmos-omnibus:v0.3.42-canto-v5.0.0`|
+|Image|`ghcr.io/akash-network/cosmos-omnibus:v0.3.42-canto-v6.0.0`|
 
 ## Examples
 

--- a/canto/build.yml
+++ b/canto/build.yml
@@ -10,7 +10,7 @@ services:
         PROJECT_DIR: .cantod
         VERSION: v6.0.0
         REPOSITORY: https://github.com/Canto-Network/Canto
-        NAMESPACE: BCNAD
+        NAMESPACE: CANTOD
     ports:
       - '26656:26656'
       - '26657:26657'

--- a/canto/build.yml
+++ b/canto/build.yml
@@ -8,7 +8,7 @@ services:
         PROJECT: canto
         PROJECT_BIN: cantod
         PROJECT_DIR: .cantod
-        VERSION: v5.0.0
+        VERSION: v6.0.0
         REPOSITORY: https://github.com/Canto-Network/Canto
         NAMESPACE: BCNAD
     ports:

--- a/canto/deploy.yml
+++ b/canto/deploy.yml
@@ -3,7 +3,7 @@ version: "2.0"
 
 services:
   node:
-    image: ghcr.io/akash-network/cosmos-omnibus:v0.3.42-canto-v5.0.0
+    image: ghcr.io/akash-network/cosmos-omnibus:v0.3.42-canto-v6.0.0
     env:
       - MONIKER=node_1
       - CHAIN_JSON=https://raw.githubusercontent.com/cosmos/chain-registry/master/canto/chain.json

--- a/canto/docker-compose.yml
+++ b/canto/docker-compose.yml
@@ -2,7 +2,7 @@ version: '3.4'
 
 services:
   node_1:
-    image: ghcr.io/akash-network/cosmos-omnibus:v0.3.42-canto-v5.0.0
+    image: ghcr.io/akash-network/cosmos-omnibus:v0.3.42-canto-v6.0.0
     ports:
       - '26656:26656'
       - '26657:26657'

